src/api/models/dataset.js

```diff
@@ -41,16 +41,19 @@ export default db => {
     collection.ensureIsUnique = async fieldName =>
         (await collection.countNotUnique(fieldName)) === 0;
 
-    collection.bulkUpdate = async (items, getFilter, upsert = false) =>
-        collection.bulkWrite(
-            items.map(item => ({
-                updateOne: {
-                    filter: getFilter(item),
-                    update: { $set: item },
-                    upsert,
-                },
-            })),
-        );
+    collection.bulkUpdate = async (items, getFilter, upsert = false) => {
+        if (items.length) {
```

Comment:
A "bulkWrite" with empty array should work, no?

Reply:
It look like it fails in mongo. But we are using 2.2 and bulkWrite is tagged as a 3.2 feature, so strange behaviour are not surprising

Reply:
OK :+1: